keps/prod-readiness/sig-api-machinery/2887.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2887
+alpha:
+  approver: "@deads2k"
```

Comment:
How come you're missing so many fields here?

Reply:
That's the PRR, scroll down a bit for the main KEP YAML.